src/lib/validators.ts

```diff
@@ -13,3 +13,25 @@ export const unixTimeFromDateString = new t.Type<number, string, unknown>(
     }),
   a => new Date(a).toISOString(),
 );
+
+const isValidEmail = (value: string): boolean => {
+  if (value === '') {
+    return true;
+  } else if (value.length > 320) {
+    return false;
```

Comment:
Very small nitpicky comment: Try to avoid using _else if_
we can just use myultiple ifs, especially when we return from it.